velox/connectors/hive/storage_adapters/s3fs/tests/S3FileSystemFinalizeTest.cpp

```diff
@@ -20,7 +20,7 @@
 
 #include "gtest/gtest.h"
 
-using namespace facebook::velox;
+namespace facebook::velox {
```

Comment:
nit: add an anonymous namespace here

Reply:
Should the tests also live in an anonymous namespace? I thought only the test class.

Reply:
the tests generate symbols, and I suppose they could conflict if symbols with the same name are defined somewhere else and linked together. Probably not super likely, but to be on the safe side, probably better.